README.md

```diff
@@ -600,6 +600,10 @@ uv accepts the following command-line arguments as environment variables:
   packages.
 - `UV_EXCLUDE_NEWER`: Equivalent to the `--exclude-newer` command-line argument. If set, uv will
   exclude distributions published after the specified date.
+- `UV_PYTHON_INSTALL_MIRROR`: Managed Python installations are downloaded from [`python-build-standalone`](https://github.com/indygreg/python-build-standalone).
```

Comment:
Sorry, but can you update `docs/configuration/environment.md` instead?

Reply:
Done